app/controllers/users_controller.rb

```diff
@@ -1,26 +1,37 @@
 # frozen_string_literal: true
 
 class UsersController < AuthorizationController
-  before_action :user_find, only: %i[show]
+  before_action :user_find, only: %i[show edit update]
 
   def show
     if @user
       render 'users/show'
     else
-      render 'users/error', status: :not_found
+      render :not_found, status: :not_found
     end
   end
 
-  def check_username
-    @user = User.find_by(name: params[:name])
-  end
+  def edit; end
 
-  def check_email
-    @user = User.find_by(email: params[:email])
+  def update
+    if params[:avatar].present? && !@user.nil?
```

Comment:
It's better to use @user.present? instead of !@user.nil?

Reply:
Done